OMFAPITests/UnitTest1.cs

```diff
@@ -101,22 +101,24 @@ private static bool Cleanup()
 
```

Comment:
While you're in here, would you mind renaming this file? This isn't actually a unit test, we should either be calling these files Tests or E2ETests imo. I could do this in a separate PR if you want to move on for now.

Reply:
I agree with this, but I think it's probably better for a separate effort since this is found in almost all of our repos. Most of our files are named UnitTest.cs, but perform E2E tests instead. Doing this separately would make the change more trackable I think.